nss-day-cohort-48/giffygram-looksmart
# Slm giffygram looksmart

#### Changes Made

1. Modified file `PostList.js` to include `PostList()` and functionality.
2. Invoked PostList() on GiffyGram.js
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout PostList.js
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    >Serve to browser and ensure that html is rendering showing the gif title, URL, and description

4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
